schema/readout-FakeCardReader-schema.jsonnet

```diff
@@ -16,6 +16,9 @@ local fakecardreader = {
     uint4  : s.number("uint4", "u4",
                      doc="An unsigned of 4 bytes"),
 
+    int8  : s.number("int8", "i8",
```

Comment:
You also need to update the corresponding jsonnet make file:
https://github.com/DUNE-DAQ/readout/blob/develop/schema/readout-FakeCardReader-make.jsonnet

And please also add the field with the default -1 to the command file:
https://github.com/DUNE-DAQ/readout/blob/develop/schema/fakereadout-app.jsonnet